elements/pfe-card/demo/pfe-card.html

```diff
@@ -1,12 +1,5 @@
-<link rel="stylesheet" href="/core/pfe-styles/red-hat-font.min.css">
```

Comment:
@bennypowers removing these stylesheets was intentional? Just want to confirm.

Reply:
It also looks like all the files under `pfe-autocomplete` were removed.

Reply:
Maybe they're just missing from the `elements/pfe.ts` file?

Reply:
yes, we're removing `pfe-styles` package. RHDS supercedes it.

as well, pfe-autocomplete, select, and dropdown are not quite ready for 1:1, so we removed them. progress-stepper should still be there

Reply:
Roger that. The rest looks good to me! :)

Reply:
Like my assigned "rest", not the "rest rest" :)